kyber/__init__.py

```diff
@@ -42,13 +43,7 @@ def deploy_app(tag, force, yes):
         tag = 'git_{}'.format(tag)
 
     if not yes:
-        deployed_app = Environment(context.name).app
-
-        click.echo("Project: {}".format(context.name))
-        click.echo("Docker: {}".format(context.docker))
-        click.echo("Deployed tag: {}".format(deployed_app.tag if deployed_app is not None else 'N/A'))
-        click.echo("Tag to be deployed: {}".format(tag))
-
+        status.echo(context)
```

Comment:
Will this not fail because `echo` excepts three params, with the second two not having defaults? Shouldn't `echo` be defined as

```
def echo(context, skip_k8s=False, skip_ecr=False):
    ...
```

?